lessons/2 - git time/index.html

```diff
@@ -1,80 +1,294 @@
+        }
+    </style>
 </head>
 
 <body>
-	<a href="https://www.redi-school.org/"><img src="../../redi-school-logo.jpg" class="logo" /></a>
-	<div class="reveal">
-		<div class="slides">
-			<section>
-				<h1>Lecture</h1>
-				<h3>Git time</h3>
-			</section>
-		</div>
-	</div>
+<a href="https://www.redi-school.org/"><img src="../../redi-school-logo.jpg" class="logo"/></a>
+<div class="reveal">
+    <div class="slides">
+        <section>
+            <h1>Lecture</h1>
+            <h3>Git time</h3>
+        </section>
+        <section>
+            <h3>Create git hub account</h3>
+            <ul>
+                <li class="fragment">Open https://github.com/</li>
+                <li class="fragment">Sign up: username, email, password</li>
+                <li class="fragment">Choose free option</li>
+                <li class="fragment">Verify email - click link in email</li>
+                <li class="fragment">Done</li>
+            </ul>
+        </section>
+        <section>
+            <h3>Create git project</h3>
+            <ul>
+                <li class="fragment">Open https://github.com/ and login if needed</li>
+                <li class="fragment">Select 'repositories' and create new one</li>
+                <li class="fragment">Name it redi_js_spring_2020</li>
+                <li class="fragment">Make it public</li>
+                <li class="fragment">Done</li>
+            </ul>
+        </section>
+        <section>
+            <h3>Invite collaborators</h3>
+            <ul>
+                <li class="fragment">Open https://github.com/</li>
+                <li class="fragment">Select your project, open settings</li>
+                <li class="fragment">Select: Manage access</li>
+                <li class="fragment">Add all teachers as collaborators:</li>
+                <ul class="fragment">
+                    <li>aberezhk, BTBTravis, caritosteph, jeffersondarcy, MrTim</li>
+                </ul>
+                <li class="fragment">Done</li>
+            </ul>
+        </section>
+        <section>
+            <h3>Download and install git</h3>
+            <ul>
+                <li class="fragment">Check if you have 64 or 32 bit system</li>
+                <li class="fragment">Open https://git-scm.com/download/win</li>
+                <li class="fragment">Download git executable (64 or 32 bit)</li>
+                <li class="fragment">Execute it</li>
+                <li class="fragment">Accept all default settings</li>
+                <li class="fragment">Done</li>
+            </ul>
+        </section>
+        <section>
+            <h3>Clone your project</h3>
+            <ul>
+                <li class="fragment">Open This PC -> C: -> create folder 'git' and open it</li>
+                <li class="fragment">Right mouse click -> open git bash here</li>
+                <li class="fragment">Open your project in git hub</li>
+                <li class="fragment">Select HTTPS and copy the link</li>
```

Comment:
This is clever so we don't have to deal with setting up SSH keys on everyone's laptops. 👍

Reply:
Added ssh setup as advanced and optional homework assignment